modules/menus/menu_actor_spawn/src/actor_spawn_menu.cpp

```diff
@@ -0,0 +1,148 @@
+#include "menus/menu_actor_spawn/include/actor_spawn_menu.h"
+#include <cstdio>
+#include "settings.h"
+#include "libtww/include/d/com/d_com_inf_game.h"
+#include "libtww/include/f_op/f_op_actor_mng.h"
+#include "libtww/include/m_Do/m_Do_printf.h"
+#include "gz_flags.h"
+#include "pos_settings.h"
+#include "rels/include/defines.h"
+#include "menus/utils/menu_mgr.h"
+#include "menus/menu_actor_list/include/actor_list_menu.h"
+#include "global_data.h"
```

Comment:
maybe we should make `proc_bin_data.cpp` and `include/proc_bin_data.h` which has the struct and `loadActorName` functions defined. This way we arent reimplementing twice. I can help with this if you want